### DISCUSSION TITLE
How can I disable rotation gesture?

### DISCUSSION THREAD

**Discussion by diego9497** [ValentinH/react-easy-crop#480] [Q&A] [answered]
In my current project, users can only rotate by 90 degrees using a button. However, I haven't figured out how to disable this gesture on mobile without disabling zoom in/out.

I think I have the idea that maybe with the property, `onTouchRequest `could be possible, but I am not sure how to determine this gesture.

Or maybe it could be made into a new prop, which disables the gesture but still takes into account the prop `rotation`

**Comment by ValentinH** [comment, marked as answer]
It's pretty easy: you just have to not implement the `onRotationChange` prop.
